src/datasets/load.py

```diff
@@ -635,6 +635,7 @@ def load_dataset(
     save_infos: bool = False,
     script_version: Optional[Union[str, Version]] = None,
     use_auth_token: Optional[Union[bool, str]] = None,
+    task: Optional[str] = None,
```

Comment:
Should we add `TaskTempplate` here too? 👀

Reply:
good catch!